deployment/Makefile

```diff
@@ -1,4 +1,6 @@
-PROJECT_ID := ford3
+export COMPOSE_PROJECT_NAME=ford3
+export COMPOSE_FILE=docker-compose.yml
```

Comment:
We will extend this to load extra `docker-compose.overrides.yml` for user/development config.